src/pages/Loans/LoansOverview/components/ApyTooltip/BreakdownGroup.tsx

```diff
@@ -0,0 +1,49 @@
+import Big from 'big.js';
+
+import { Dd, Dl, DlGroup, Dt } from '@/component-library';
+
+import { getApyLabel } from '../../utils/apy';
+import { StyledApyTooltipGroup, StyledApyTooltipTitle } from './ApyTooltip.style';
+
+type BreakdownGroupProps = {
+  assetApy: Big;
+  assetTicker: string;
+  rewardsApy?: Big;
+  rewardsTicker?: string;
+  isBorrow: boolean;
+};
+
+const BreakdownGroup = ({
+  assetApy,
+  rewardsApy,
+  assetTicker,
+  rewardsTicker,
+  isBorrow
+}: BreakdownGroupProps): JSX.Element => {
+  const assetApyLabel = getApyLabel(assetApy);
+
+  return (
+    <DlGroup direction='column' alignItems='flex-start' gap='spacing1'>
+      <StyledApyTooltipTitle>APY Breakdown</StyledApyTooltipTitle>
+      <Dt>
+        <Dl direction='column' alignItems='flex-start' gap='spacing0'>
+          <StyledApyTooltipGroup gap='spacing1' wrap>
+            <Dd color='tertiary'>
+              {isBorrow ? 'Borrow' : 'Lend'} APY {assetTicker}:
+            </Dd>
+            <Dt color='primary'>{assetApyLabel}</Dt>
+          </StyledApyTooltipGroup>
+          {!!rewardsApy && (
+            <StyledApyTooltipGroup gap='spacing1' wrap>
+              <Dd color='tertiary'>Rewards APY {rewardsTicker}:</Dd>
+              <Dt color='primary'>{getApyLabel(rewardsApy)}</Dt>
```

Comment:
Displayed texts should be placed into translations so it's more maintainable. But probably we need to do this for all loans components?

Reply:
Yeah, this needs to be done across Loans